public/js/app/pages/CourseMemo.jsx

```diff
@@ -212,13 +210,8 @@ function CourseMemo() {
           />
           <Row>
             <Col id="flexible-content-of-center" lg="8" className="text-break col-print-12 content-center">
-              <CoursePresentation
-                courseImageUrl={courseImageUrl}
-                introText={webContext.sellingText}
-                labels={coursePresentationLabels}
-              />
               <p className="mb-4">
-                {sectionsLabels.asterisk} {seasonStr(i18n.messages[memoLanguageIndex].extraInfo, validFromTerm)}
+                {sectionsLabels.asterisk} {seasonStr(extraInfo, validFromTerm)}
               </p>
```

Comment:
I realised that we got the data for "extraInfo" in previous steps so I decided to make this a little shorter in case of having cleaner code